base/400-pipeline0-task.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```

Comment:
we'll need to update the spec too:

```yaml
spec:
  resources:
    inputs:
      - name: git-source
        type: git
  params:
    ...
```

Reply:
Yep, thanks, I noticed that from your change https://github.com/tektoncd/dashboard/commit/080a81ef493f0e3780c5e1ffeea287fdff32f957 just now actually 🤞